src/osc-bridge/schema/test.json

```diff
@@ -634,8 +634,8 @@
         "tooltip"  : "Attack Time",
         "scale"    : "logarithmic",
         "type"     : "f",
-        "range"    : [0.0f,41.0f],
-        "default"  : "0.0f"
+        "range"    : [0.0001f,41.0f],
```

Comment:
the github highlighting makes me think this should be `[0.0001,41.0]` i.e. without the 'f'.